FreeRTOS/FreeRTOS-Kernel
# Print task priority value in vTaskGetRunTimeStats()

Show task priority for better understanding of task run time consumption
with context of task priority



Description
-----------
I found it useful to observe task priority in the context of run time consumption.
This modification is platform independent.

Test Steps
-----------
I did simple test just run this code in my environment after task name it is priority number
IDLE           	0		265862		59%
Tmr Svc        	6		427		<1%
UART       	1		942		<1%

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
